tests/unit/test_arguments.py

```diff
@@ -23,3 +26,36 @@ def test_can_set_required(self):
         self.assertFalse(arg.required)
         arg.required = True
         self.assertTrue(arg.required)
+
+
+class TestCLIArgument(unittest.TestCase):
+    def setUp(self):
+        self.arg_name = 'foo'
+        self.op_name = 'bar'
+        self.service_name = 'baz'
+        self.argument_model = mock.Mock()
```

Comment:
I would prefer to either not have this test or to use real argument_model objects.  I don't think there's a lot of value in mocking out the model objects.